JPS_BASE_LIB/src/test/java/uk/ac/cam/cares/jps/base/derivation/DerivationOutputsTest.java

```diff
@@ -579,9 +579,9 @@ public String formulateLiteralTripleString(String s, String p, Object o) {
 			if (o instanceof Double) {
 				if (((Double) o).isInfinite()) {
 					if (((Double) o) > 0) {
-						return "<" + s + "> <" + p + "> \"INF\"^^<" + XSD.DOUBLE.toString() + "> .";
+						return "<" + s + "> <" + p + "> \"Infinity\"^^<" + XSD.DOUBLE.toString() + "> .";
```

Comment:
This is being nitpicky and I also used to do this, but in the future, please use something like String.format("<%s> <%s> \"o\"^^<%s>", s, p, o, type) to improve readability